src/libs/print_settings.c

```diff
@@ -0,0 +1,1398 @@
+  d->printers = comb;
+  g_signal_connect(G_OBJECT(d->printers), "changed", G_CALLBACK(_printer_changed), self);
+  g_list_free (printers);
+
+  //  Add printer profile combo
+
+  d->pprofile = dt_bauhaus_combobox_new(NULL);
+  dt_bauhaus_widget_set_label(d->pprofile, NULL, _("profile"));
+
+  int combo_idx = -1, n=0;
+  GList *l = d->profiles;
+
+  gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(d->pprofile), TRUE, TRUE, 0);
+  const gchar *printer_profile = dt_conf_get_string("plugins/print/printer/iccprofile");
+  combo_idx = -1;
+  n=0;
+
+  dt_bauhaus_combobox_add(d->pprofile, _("none"));
+  while(l)
+  {
+    dt_lib_export_profile_t *prof = (dt_lib_export_profile_t *)l->data;
+    // do not add built-in profile, these are in no way for printing
+    if (strcmp(prof->filename,"sRGB")!=0 && strcmp(prof->filename,"adobergb")!=0)
+    {
+      dt_bauhaus_combobox_add(d->pprofile, prof->name);
+      n++;
+      if (strcmp(prof->filename,printer_profile)==0)
+        combo_idx=n;
+    }
+    l = g_list_next(l);
+  }
+
+  // profile not found, maybe a profile has been removed? revert to none
+  if (combo_idx == -1)
+  {
+    dt_conf_set_string("plugins/print/printer/iccprofile", "");
+    combo_idx=0;
+  }
+  dt_bauhaus_combobox_set(d->pprofile, combo_idx);
+
+  snprintf(tooltip, sizeof(tooltip), _("printer ICC profiles in %s/color/out or %s/color/out"), confdir, datadir);
+  g_object_set(G_OBJECT(d->pprofile), "tooltip-text", tooltip, (char *)NULL);
+  g_signal_connect(G_OBJECT(d->pprofile), "value-changed", G_CALLBACK(_printer_profile_changed), (gpointer)self);
+
+  //  Add printer intent combo
+
+  d->pintent = dt_bauhaus_combobox_new(NULL);
+  dt_bauhaus_widget_set_label(d->pintent, NULL, _("intent"));
+  dt_bauhaus_combobox_add(d->pintent, _("perceptual"));
+  dt_bauhaus_combobox_add(d->pintent, _("relative colorimetric"));
+  dt_bauhaus_combobox_add(d->pintent, C_("rendering intent", "saturation"));
+  dt_bauhaus_combobox_add(d->pintent, _("absolute colorimetric"));
+  gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(d->pintent), TRUE, TRUE, 0);
+  dt_bauhaus_combobox_set(d->pintent, dt_conf_get_int("plugins/print/printer/iccintent"));
+
+  g_signal_connect (G_OBJECT (d->pintent), "value-changed", G_CALLBACK (_printer_intent_callback), (gpointer)self);
+
+  ////////////////////////// PAGE SETTINGS
+
+  label = dt_ui_section_label_new(_("page"));
+  gtk_box_pack_start(GTK_BOX(self->widget), label, TRUE, TRUE, 0);
+
+  //// papers
+
+  dt_bauhaus_widget_set_label(d->papers, NULL, _("paper"));
+
+  g_signal_connect(G_OBJECT(d->papers), "value-changed", G_CALLBACK(_paper_changed), self);
+  gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(d->papers), TRUE, TRUE, 0);
+
+  // now set recorded default/active printer
+
+  gtk_combo_box_set_active(GTK_COMBO_BOX(comb), printer_index);
+  _set_printer(self, printer_name);
+
+  //// portrait / landscape
+
+  d->landscape = gtk_radio_button_new_with_label(NULL, _("landscape"));
```

Comment:
Use a bauhaus combobox instead, label "orientation" and the two options "landscape" and "portrait".